pkg/cloudprovider/vsphere/nodemanager.go

```diff
@@ -652,3 +671,59 @@ func (nm *NodeManager) getNodeNameByUUID(UUID string) string {
 	}
 	return ""
 }
+
+func extractGuestInfo(extraConfig []types.BaseOptionValue) (string, string) {
```

Comment:
nit: I think it could be more accurate if renamed to `guestInfoMetadata`

Reply:
We've renamed it, thanks for the review!